app/components/spin-button.js

```diff
@@ -90,16 +91,19 @@ export default Ember.Component.extend({
       this.setDisabled();
 
       if (this.get('startDelay') > 4) {
-        Ember.run.later(this, this.createSpinner, element, this.get('startDelay'));
+       this.set('_spinnerTimer', Ember.run.later(this, this.createSpinner, element, this.get('startDelay')));
       }else{
         this.createSpinner(element);
       }
     }else{
+      Ember.run.cancel(this.get('_spinnerTimer'));
+      this.set('_spinnerTimer', null);
       this.setEnabled();
     }
   }.observes('inFlight'),
 
   createSpinner: function(element) {
+    this.set('inFlightStyle', true);
```

Comment:
I pushed a fix, but it just renamed the _inFlightStyle_ property to something more meaningful... Why do you think it is a breaking change? I guess, this is just an internal change that does not affect the end-user.

The problem I was trying to fix by this is that _inFlight_ property should not be used in a _classNameBindings_ - because setting _inFlight_ to true does not necessarily mean, that spinner should be shown